core/src/style/pointStyle.h

```diff
@@ -50,6 +53,8 @@ class PointStyle : public Style {
 
     UniformLocation m_uTex{"u_tex"};
     UniformLocation m_uOrtho{"u_ortho"};
+
+    mutable std::unique_ptr<LabelMesh> m_mesh;
```

Comment:
I think names are fine for now, we can/will revisit these later. It will be good to have comments in the header, describing these.